packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -50,6 +51,10 @@
 
 public class NodesManager implements EventDispatcherListener {
 
+  private Long mFirstUptime = SystemClock.uptimeMillis();
+  private boolean mSlowAnimationsEnabled = false;
+  private long mAnimationsDragFactor;
```

Comment:
Why do we have this as a variable if we only ever pass the hard-coded value from `NativeProxyCommon.java` here

Reply:
+1 on this.

Reply:
I made this change because `ANIMATIONS_DRAG_FACTOR` is also utilized in the `NativeProxyCommon::getAnimationTimestamp()` method. I wanted to define it in only one place and pass it as an argument here.